Package.swift

```diff
@@ -124,7 +124,10 @@ let package = Package(
         .product(name: "SwiftParser", package: "swift-syntax"),
         .product(name: "SwiftToolsSupport-auto", package: "swift-tools-support-core"),
       ],
-      exclude: ["CMakeLists.txt"]
+      exclude: ["CMakeLists.txt"],
+      swiftSettings: [
+        .enableUpcomingFeature("BareSlashRegexLiterals")
+      ]
```

Comment:
Sorry, forgot that that’s needed to make it compile in Swift 6 mode. Thanks for adding it. 🙏🏽

Reply:
😌